core/app/waste_item/application/use_cases/get_objets_recycling_material.py

```diff
@@ -0,0 +1,15 @@
+from core.app.waste_item.domain.ports import WasteItemRepository
+
+class GetObjetsRecyclingMaterialUseCase:
+    def __init__(self, waste_item_repository: WasteItemRepository):
+        self.waste_item_repository = waste_item_repository
```

Comment:
Naming is not accurate, try something like

```suggestion
class CountMaterialAmountUseCase:
    def __init__(self, waste_item_repository: WasteItemRepository):
        self.waste_item_repository = waste_item_repository
```